content/content-strategy/manage-content-requests/create-content/pair-writing/how.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: text/textblock
+section: How to do pair writing
+---
+### [1]Step 1: Start with user needs
+
+At the start of the session you need to outline your user needs and the aim of the content. The aim may include a business need or the subject expert’s objective for the content.
+
+Articulating the user needs at the start will put you both in the same mindset, working towards the same goal.
+
+If you are in a workshop it’s useful to get group feedback on your user needs and content objective.
+
+### [1]Step 2: Outline your roles
+
+One person will be the main writer and the other is the observer. The observer’s role is to study and question the content, looking at:
+
+- jargon — is there a clearer way of writing it?
+- user needs — does the content meet user needs?
+- meaning — is the content clear?
+- style — does the content follow the Content Guide?
+
+You should swap roles so you both experience writing and observing.
+
+### [1]Step 3: Create the content
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3.

Reply:
Trisha: done